components/Dialogs/src/private/MDCDialogTransitionMotionSpec.m

```diff
@@ -37,7 +37,8 @@ + (MDCDialogTransitionAppearanceTimings)appearance {
     .contentScale = {
       .duration = 0.150, .delay = 0.000, .curve = [self deceleration]
     },
-    .contentScaleFromValue = 0.8
+    .contentScaleFromValue = 0.8,
+    .transitionDuration = 0.150,
```

Comment:
It would be nice if this could be some kind of "meta property" on a timing NSObject. One where it would inspect all properties of type MDMTiming and return the MAX of all of their `duration + delay`. Something uniform like that.

Reply:
Agreed. There are an increasing number of arguments for making these specs be true Objective-C objects, the other argument being around the eventual live tooling/tweaking interfaces we'd like to provide. Most of these use cases require some form of introspection.

Reply:
I'll have to explore that separately from this PR as a general improvement to our motion spec'ing infrastructure though.